src/kaiteki/assets/instances.json

```diff
@@ -288,16 +288,19 @@
     "shortDescription": "Fun hangout for furries and normies alike.",
     "favicon": "https://stop.voring.me/assets/about-icon.86c5c1fe.png",
     "rules": [
-      "None of the following:",
+      "Please read the whole TOS for the full rules list 🙇",
+      "Must be at least 13 years of age",
+      "If you are under 18, no NSFW content must be viewed or shared. All NSFW must be CW'd.",
+      "Try and be nice to everyone else!",
+      "None of the following is allowed:",

```

Comment:
The rules list may only include rules and no headings or whatsoever, it isn't displayed as simple text.